src/js/graph/node.js

```diff
@@ -76,6 +79,134 @@ Node.prototype.hasTraitSpanContainer = function() {
+                break;
+            }
+        }
+    }
+};
+Node.prototype._connectEdgesSequence = function() {
+    var self = this;
+    if (!self.hasTraitSequence()) { return; }
+    // Sequence (first)
+    // Handled by the generic edge loop above.
+    // Sequence (middle)
+    // A -> B
+    // A C -> B
+    // A -> C -> B
+    if (self.hasNext() && !self.hasPrevious() &&
+        self.next().hasPrevious() &&
+        self.next().previous() !== self
+    ) {
+        var prevNode = self.next().previous();
+        prevNode.removeEdge(prevNode.getFirstEdgeByType('next'));
+        prevNode.addEdge(self.getId(), self.getType(), 'next');
+        // Next node is already added via the generic edge loop above.
+        // Set parent edges based on this node's neighbors
+        var prevInEdgeImpls = prevNode._getEdgesImplByLabels('in', ['first', 'last', 'child']);
+        var nextInEdgeImpls = self.next()._getEdgesImplByLabels('in', ['first', 'last', 'child']);
+        // Matching parents between the prev and next nodes will be applied to this node.
+        prevInEdgeImpls.forEach(function(prevInEdgeImpl) {
+            nextInEdgeImpls.forEach(function(nextInEdgeImpl) {
+                if (prevInEdgeImpl.v == nextInEdgeImpl.v) {
+                    var prevParentNode = self._graph.getNodeById(prevInEdgeImpl.v);
+                    // console.log("LINK EDGE nodeId: " + prevParentNode.getId() + ", targetId: " + self.getId() + ", type: child.");
+                    self._graph._graphImpl.setEdge(prevParentNode.getId(), self.getId(), 'child');
+                }
+            });
+        });
+    }
+    // Sequence (last)
+    // Is this even possible? There are no edges to indicate that it is.
+};
+Node.prototype.removeEdge = function(aEdge) {
+    if (!(aEdge instanceof Edge)) {
+        throw Error("`Node.removeEdge` only accepts an instance of Edge.");
+    }
+    // console.log("REMOVE EDGE nodeId: " + this.getId() + ", targetId: " + aEdge.getTargetId() + ", targetType: " + aEdge.getTargetType() + ", edgeType: " + aEdge.getEdgeType() + ".");
+    var self = this;
+    self._edges = self._edges.filter(function(edge) {
+        var matches = aEdge === edge;
+        var edgeNode = self._graph.getNodeById(edge.getTargetId());
+
+        // Remove the edge from the graphImpl.
+        // console.log("UNLINK EDGE nodeId: " + self.getId() + ", targetId: " + edgeNode.getId() + ", type: " + edge.getEdgeType() + ".");
+        self._graph._graphImpl.removeEdge(self.getId(), edgeNode.getId(), edge.getEdgeType());
+        // Only keep edges that don't match the one provided.
+        return aEdge !== edge;
+    });
+};
+Node.prototype.removeEdges = function() {
+    var self = this;
+    self._removeEdgesSpanContainer();
+    self._removeEdgesSequence();
+    // Remove the standard edges
+    self.getEdges().forEach(function(edge) {
+        self.removeEdge(edge);
+    });
+};
+Node.prototype._removeEdgesSpanContainer = function() {
```

Comment:
Unless this is handled by the loop on line 185, this function needs to be written. If it is handled in the generic loop, lets delete it and remove the call from line 182.